test_spec_PY3

```diff
@@ -0,0 +1,6 @@
+# Excludes
+- annie
```

Comment:
Why is annie excluded?

Reply:
Most of the tests were broken up until yesterday's release.  Had a plan to remove the exclude.